zuul-core/build.gradle

```diff
@@ -47,7 +47,7 @@ dependencies {
     runtimeOnly( group: "io.netty", name: "netty-tcnative-boringssl-static", classifier: "osx-aarch_64" )
 
     implementation 'io.perfmark:perfmark-api:0.26.0'
-    implementation 'javax.inject:javax.inject:1'
+    api 'jakarta.inject:jakarta.inject-api:2.0.1'
```

Comment:
Change to `api` is necessary to get `jakarta.inject-api` into internal zuul's classpath.